integration/v3_lease_test.go

```diff
@@ -438,6 +440,8 @@ func TestV3LeaseFailover(t *testing.T) {
 		t.Fatal(lresp.Error)
 	}
 
+	expectedExp = time.Now().Add(time.Duration(lresp.TTL) * time.Second)
```

Comment:
> lease proxy uses watch to detect leader loss where as the etcd server has some different logic for that. 

They both go through the same interceptor path on the etcd server; the only difference is there is a separate stream monitoring for leader loss on the proxy. Why is the etcd server processing a keepalive request/response before erroring out on leader loss, but not the proxy? There should still be a delay between the leader loss and the etcd server recognizing that loss...

This needs some kind of investigation instead of trying to mask it.